README.md

```diff
@@ -125,6 +125,7 @@ List of projects that provide terminal user interfaces
 <details open><summary><h2>File Managers</h2></summary>
 
 - [adbtuifm](https://github.com/darkhz/adbtuifm) A TUI file manager for Android, based on the Android Debug Bridge(ADB).
+- [broot](https://github.com/Canop/broot) A fuzzy finder file manager written in Rust.
```

Comment:
Maybe the part about  tree navigation mentioned in previous entry could be kept here